server/utils/drupal/drupal-forums.js

```diff
@@ -36,12 +43,13 @@ export async function getForumTidFromAlias(ctx){
     return parseInt(path.replace('/taxonomy/term/',''))
 }
 
-async function getForums(ctx) {
+async function getForums(ctx, all=false) {
 
     const { host, uuid, localizedHost  } = ctx;
-    const id            = uuid? `/${uuid}` : '';
+    const id            = uuid && !all? `/${uuid}` : '';
+
+    const params        = getQuestString(ctx, all);
 
-    const params        = getQuestString(ctx);
     const uri           = `${localizedHost}/jsonapi/taxonomy_term/forums${encodeURIComponent(id)}?jsonapi_include=1${params}`;//&include=taxonomy_forums&page[limit]=${rowsPerPage}&sort=-sticky
```

Comment:
are you missing slah?
```suggestion
    const uri           = `${localizedHost}/jsonapi/taxonomy_term/forums/${encodeURIComponent(id)}?jsonapi_include=1${params}`;//&include=taxonomy_forums&page[limit]=${rowsPerPage}&sort=-sticky
```